README.md

```diff
@@ -145,11 +148,42 @@ The following instructions describe how to set up the pet clinic sample applicat
    java -jar spring-petclinic-api-gateway...
    ```
 
-6. Repeat step 5 for the remaining EC2 instances (vets, customers, visits)
+6. Repeat step 5 for the EC2 instances (vets, customers, visits)
 
-7. Visit the sample application by going to `http://<PUBLIC-IPv4-DNS-OF-PET-CLINIC-FRONTEND-INSTANCE>:8080`
+7. Go to RDS and create a Postgres DB with the following configurations:
+    - Use the Dev/Test template with a single DB instance. 
+    - Update the Master username to `root` and create a password of your choosing. Write it down since you will need it later. 
+    - In the Connectivity settings, use the VPC, public subnet, and security group created in step 1. 
+    - Switch to Connect to an EC2 compute instance and choose the vehicle-service EC2 instance and then create the DB.
 
-8. Interact with the application to ensure you've properly set up the backend services. Note that each service takes a few seconds to come up.
+8. Select the EC2 instance names `insurances` and choose Actions -> Networking -> Connect RDS and choose the RDS instance from step 7. 
+
+9. Connect to the EC2 instance named `insurances` and then run the following commands to start the python microservice in a tmux session. Feel free to end your connection to the EC2 instance, the tmux sessions will continue running.
+   - Create a zip file of the `pet_clinic_insurance_service` directory and upload it to S3 bucket `s3://app-signals-ec2-demo`.
+   - Download the zip file from S3 bucket
+   ```
+   aws s3 cp s3://app-signals-ec2-demo-asakem/pet_clinic_insurance_service.zip .
+   unzip pet_clinic_insurance_service.zip
+   cd pet_clinic_insurance_service
+   . ec2-setup.sh <PASS-FROM-STEP-7> <PRIVATE-IP-OF-SETUP-INSTANCE>
+   ```
+
+   - Run the sample app in a tmux session and then exit by inputting `ctrl+b, d`.
+   ```
+   tmux new -s insurance
+   python3.11 manage.py migrate & python3.11 manage.py loaddata initial_data.json & python3.11 manage.py runserver 0.0.0.0:8000 --noreload
```

Comment:
This is because in EC2, we install python3.11 on the instance and that's what is available for EC2 through yum. Once installed, we need to use python3.11.

Reply:
Will this break when the python version is updated on EC2? I don't understand why can't we just use `python3` assuming compatibility between minor versions.

Reply:
I just tested it again. python3 should work fine. I think i ran into issues previously that caused me to install python3.11 but it shouldn't matter.